src/db_mgr.cc

```diff
@@ -21,6 +21,7 @@ limitations under the License.
 #include "db_internal.h"
 #include "flusher.h"
 #include "internal_helper.h"
+#include "libjungle/db_config.h"
```

Comment:
Can you move this after `log_reclaimer.h` and before `<set>`? Should be treated as 3rd party header.

Reply:
Fixed.